src/apiCalls/api.js

```diff
@@ -0,0 +1,12 @@
+const getData = (endpoint) => {
+   return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/${endpoint}`)
+    .then(response => {
+      if(response.ok) {
+         return response.json()
+        } else {
+          throw new Error("Sorry! Failed to load, please try again later.")
+        }
```

Comment:
Probably a good idea to include a .catch here - that would give us the exact error from API, which we could then store in state and display to the user :)